### PR TITLE
Implement category radio button

### PR DESCRIPTION
**Pull request by Kajochi**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM openjdk:20
+
+ENV ENVIRONMENT=prod
+
+LABEL maintainer="stefan@neuefische.de"
```
Comment (CananOzyol) [inline Dockerfile:5]: change maintainer name

File: backend/src/main/java/de/teamprojectfinance/backend/PostingService.java [hunk 2]
```diff
@@ -15,7 +15,7 @@ public PostingService(PostingRepo postingRepo, IdService idService){
     }
 
     public PostingModel addPosting(NewPostingModel newPostingModel) {
```
Comment (CananOzyol) [inline backend/src/main/java/de/teamprojectfinance/backend/PostingService.java:17]: did you consider having just one PostingModel class.

**Review by CananOzyol — APPROVED** [review]
LGTM

File: backend/src/main/java/de/teamprojectfinance/backend/PostingService.java (hunk 2 above)
Comment (Kajochi) [reply backend/src/main/java/de/teamprojectfinance/backend/PostingService.java:17]: we have used a DTO here. This way we give the user or frontend access only to certain fields of the object. Here for example the ID. This should not be changed.

**Pull request merged**